### PR TITLE
docs(index.js): modified the copy of the index.js

### PR DESCRIPTION
**Pull request by cristian-sbardelotto**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
 /* 
-    33 JavaScript Concepts is a project created to help JavaScript developers master their skills. It is a compilation of fundamental JavaScript concepts that are important and fundamental. 
+    33 JavaScript Concepts is a project created to help JavaScript developers master their skills. 
+    
+    It is a compilation of fundamental JavaScript concepts that are important and fundamental. 
 
-    This project was inspired by an article written by Stephen Curtis. 
+    JavaScript is a powerful and a very simple language. This language is used to make tasks automation, Web Pages, Servers, Desktop Apps, Databases, etc.
```
Comment (ParthJadhav) [inline index.js:6]: JavaScript is a robust and easy-to-learn language. These include Web pages, servers, desktop applications, databases, and task automation.

File: index.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+    
+    Learning the fundamental JS concepts is very important to know how to use this so strong language.
```
Comment (ParthJadhav) [inline index.js:8]: It's crucial to understand the foundational JS ideas in order to use this powerful language.

**Review by ParthJadhav — COMMENTED** [review]
Left some comments @cristian-sbardelotto

**Comment by cristian-sbardelotto** [comment]
I don't know what you mean, would these messages be suggestions to change?

Em ter., 18 de out. de 2022 às 15:28, Parth Jadhav ***@***.***>
escreveu:

**Comment by ParthJadhav** [comment]
Yes @cristian-sbardelotto , Sorry for incomplete information. They are meant to be suggestions. Feel free to modify/reject them.

**Comment by cristian-sbardelotto** [comment]
Sure, I'll add all the suggestions, your way is better

Em qua., 19 de out. de 2022 às 16:00, Parth Jadhav ***@***.***>
escreveu:

**Comment by cristian-sbardelotto** [comment]
Thanks for the help!

Em qua., 19 de out. de 2022 às 18:30, Cristian Keiber Sbardelotto <
***@***.***> escreveu:

**Comment by cristian-sbardelotto** [comment]
Thank you for your help

Em qua., 19 de out. de 2022 às 18:30, Cristian Keiber Sbardelotto <
***@***.***> escreveu:

**Pull request closed without merging**